Cargo.toml

```diff
@@ -20,6 +20,13 @@ publish = false
 [package.metadata.docs.rs]
 rustdoc-args = [ "--html-in-header", "katex-header.html" ]
 
+[dev-dependencies]
+criterion = "0.3"
+
+[[bench]]
+name = "plonk"
```

Comment:
Probably should rename this to `arithmetic` or something since it's not actually testing plonk itself.